cmd/influx2cortex/main.go

```diff
@@ -36,7 +36,7 @@ func Run() error {
 
 	recorder := influx.NewRecorder(prometheus.DefaultRegisterer)
 
-	httpAuthMiddleware := fakeauth.SetupAuthMiddleware(&serverConfig, enableAuth, nil)
+	httpAuthMiddleware := middleware.AuthenticateUser
```

Comment:
does this need to be defined out here? It's only accessed in the api.Register call so it seems like it's not actually needed as a pass-in parameter